sigar_windows.go

```diff
@@ -141,7 +141,42 @@ func (self *FileSystemList) Get() error {
 }
 
 func (self *DiskList) Get() error {
-	return notImplemented()
+	diskQueries := []string{
+		`\physicaldisk(*)\disk reads/sec`,
+		`\physicaldisk(*)\disk read bytes/sec`,
+		`\physicaldisk(*)\% disk read time`,
+		`\physicaldisk(*)\disk writes/sec`,
+		`\physicaldisk(*)\disk write bytes/sec`,
+		`\physicaldisk(*)\% disk write time`,
+	}
+
+	// Run a PDH query for metrics across all physical disks
+	queryResults, err := runRawPdhArrayQueries(diskQueries)
+	if err != nil {
+		return err
+	}
+
+	self.List = make(map[string]DiskIo)
+	for disk, counters := range queryResults {
+		if disk == "_Total" {
+			continue
+		}
+
+		self.List[disk] = DiskIo {
+			ReadOps: uint64(counters[0]),
+			ReadBytes: uint64(counters[1]),
+			// `% disk read time` is measured in 100ns ticks, 
+			// divide by 10000 to get milliseconds from the raw counter
+			ReadTimeMs: uint64(counters[2]/10000),
```

Comment:
I'll add this to the code as well:

The metric is called `% disk read time`, but PDH exposes both "raw" and "cooked" counters. The "cooked" counter takes two data points with timestamps and does the % calculation for you. The "raw" value underlying that counter is the total time spent on disk operations since the system started, measured in 100ns ticks. 

To line up with how we report disk IO on Linux (in SIGAR), I've chosen to take the raw counter underneath. So this is just a counter of how many milliseconds we've spent on reads since startup. The agent is responsible for grabbing two measurements, subtracting them, and dividing by total time to get the % utilization.